source/release-notes/4-2-x-breaking-changes.rst

```diff
@@ -19,6 +19,28 @@ before upgrading. Please review the following and apply changes as required:
 
 |hr|
 
+------------------------------------
+Breaking Changes in CrafterCMS 4.2.2
+------------------------------------
+CrafterCMS uses TinyMCE version 7 for the RTE control.
+The following properties were removed from the RTE control in CrafterCMS version 4.2.2:
+
+- ``forceRootBlockPTag``
```

Comment:
This was the internal variable name, not really something users would see. Not even developers unless they were explicitly working with the RTE code. Where this is more visible in the system is in the Type Editor.
![image](https://github.com/user-attachments/assets/a6387944-95c4-4bcf-9689-8473c9b52fac)

Let's use regular labels like shown in the picture.